scripts/irods/test/test_iadmin.py

```diff
@@ -486,6 +486,20 @@ def test_iadmin_mkuser(self):
         self.admin.assert_icommand("iadmin mkuser lando invalid_user_type",
                                    'STDERR_SINGLELINE', "CAT_INVALID_USER_TYPE")  # should be rejected
 
+    @unittest.skipIf(IrodsConfig().version_tuple < (4, 3, 2), "Only applies to 4.3.2 and above.")
+    def test_new_clients_cannot_rmuser_groups_nor_rmgroup_users__issue_7380(self):
```

Comment:
`new_clients` is a little misleading. Consider rephrasing.

Could be something like ... `test_iadmin_rmuser_rmgroup_report_error_on_mismatch_entity_type__issue_7380`.